controllers/backup/suite_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package pulp_backup
+package pulp_backup_test
```

Comment:
Not really an issue, but there is a convention on using the package name with the same name of its source dir:
https://go.dev/doc/effective_go#names

>  Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64.